src/app/toolbar/toolbar.component.html

```diff
@@ -15,21 +15,24 @@
 -->
 
 <mat-toolbar class="toolbar">
-  <mat-toolbar-row class="toolbar-row">
+  <mat-toolbar-row class="toolbar-row" [ngClass]="isMobileMenuOpen ? 'hidden-nav': ''  ">
     <a routerLink="/home" class="title-wrapper">
       <div class="icon-placeholder"></div>
       <span class="title">SafeNet</span>
     </a>
-    <ng-container *ngIf="!userIsSignedIn">
-      <span class="spacer"></span>
-      <button mat-button class="action-button get-started-button" routerLink="/">Get Started</button>
-    </ng-container>
+
+
+    <button mat-button class="action-button mobile-menu-button" (click)="toggleMobileMenu()">
+      <mat-icon svgIcon="hamburger_menu"></mat-icon>
+    </button>
+
   </mat-toolbar-row>
 
-  <mat-toolbar-row class="toolbar-row toolbar-row-signed-in" *ngIf="userIsSignedIn">
+  <mat-toolbar-row class="toolbar-row main-nav toolbar-row-signed-in" *ngIf="userIsSignedIn"
+    [ngClass]="isMobileMenuOpen ? 'hidden-nav': ''  ">
     <div class="nav-links nav-item">
       <a routerLinkActive="active-router-link" routerLink="/home" #homeLink role="link"
-        [attr.aria-current]="homeLink.routerLinkActive ? 'page' : false">
+        [attr.aria-current]="homeLink.routerLinkActive ? 'page' : false" class="home-link">
```

Comment:
```suggestion
        [attr.aria-current]="homeLink.routerLinkActive ? 'page' : false">
```
Are we using `home-link` anywhere?